### ISSUE TITLE
Material You design

### ISSUE THREAD

**Issue by BluePixel4k** [edodson84/Aosp_Dialer_10_Gradle#3]
Thank you, for publishing the AOSP Dialer for everyone. 😊 

Do you know if it would be hard to adapt the Dialer app to the Material You Guidelines?

Like this (from the Google Phone app):
<img src="https://9to5google.com/wp-content/uploads/sites/4/2021/09/google-phone-material-you-favorites.png" width="250" /> <img src="https://9to5google.com/wp-content/uploads/sites/4/2021/09/google-phone-material-you-voicemail-dark.png" width="250" />
Source: [xda-developers.com](https://www.xda-developers.com/google-phone-app-pick-up-material-you-theming-android-12/)

**Comment by edodson84** [comment]
i can look into it

**Comment by BluePixel4k** [comment]
Thx :)

**Comment by edodson84** [comment]
check out version 2.0

**Comment by BluePixel4k** [comment]
Thank you, that's a great beginning! 🚀 

It's possible to increase the bottom column a bit?
<img src="https://user-images.githubusercontent.com/77979781/183479672-06c87969-e81b-46eb-8295-c09ba8708171.png" width="300" />

Maybe you can edit the numpad a bit? 
Here is a suggestion from me (darker and with bubbles):
<img src="https://user-images.githubusercontent.com/77979781/183483558-b5b060ea-169c-405e-a610-7ccc8be71464.png" width="300" />

And a suggestion for the favorites:
<img src="https://user-images.githubusercontent.com/77979781/183486683-d91e34c2-69f2-43d5-af82-6e106d15f341.png" width="300" />

Furthermore, is it possible to include the Material You color scheme? 😃

**Comment by edodson84** [comment]
i will c what i can do alot of code would need changed to utilize material you, but i can prob make most things material 3 themed just not dynamic colors

**Comment by gigisforza70** [comment]
very nice, for a long time I was looking for an open source dialer with these features. I wanted to tell you if you can fix things with the clear theme I am sending you some photo. Practically in the light theme the text of the status bar is not read and then the button that opens the keypad is red, instead it should be blue like the dark theme. And then in the settings I would like the blue box to be white for the light theme and dark for the dark theme
![photo_6041907319543019829_y](https://user-images.githubusercontent.com/55690014/184889542-b3ea9ccb-0c64-4602-99c6-53df2d0c422d.jpg)
![photo_6041907319543019825_y](https://user-images.githubusercontent.com/55690014/184889667-aa5e90b1-9a0d-4ed4-8d9d-0d9360d49fe1.jpg)
s

**Comment by edodson84** [comment]
yes i will fix the theming issues, i was concentrating more on the dark theme i nvr liked light theming lol

**Comment by BluePixel4k** [comment]
@edodson84 Thanks for the further changes, looks great! 👍🏼 
But with the new num pad, I sadly don't harmonize. Maybe you can customize it more?

Here is another suggestion, inspirierte by the lock in screen from Android 12. Compared to the others:
<img src="https://user-images.githubusercontent.com/77979781/185073852-a8e1af42-da78-4cfc-8032-9032c58ba9b3.png" width="200" /> <img src="https://user-images.githubusercontent.com/77979781/185075213-0c5af405-4009-49e7-8799-ed0085351a7d.png" width="200" /> <img src="https://user-images.githubusercontent.com/77979781/185073864-0c89b31d-c2f3-405f-a571-25c9ad112a25.png" width="200" /> 

And can you pull the bottom column apart a bit and maybe adjust the colors a bit?
<img src="https://user-images.githubusercontent.com/77979781/185079636-56ae060b-e674-4775-b63a-1ec20ea09b4a.png" width="300" />
